src/components/Assets/AssetTable.js

```diff
@@ -29,8 +37,17 @@ const AssetTable = ({ fields, data }) => {
             data.map((record, i) => (
               <tr key={i}>
                 {fieldIds.map((fieldId, i) => {
-                  const tableDataString = record[fieldId];
-                  return handleTableDataStringLength(tableDataString, i);
+                  if (fieldId.match(/(-detector-link)/)) {
```

Comment:
It might be helpful to add a comment documentation above this `if` block. Just wasn't immediately understandable to me from the code why you need to do the match and replace.

Reply:
✅ This is a great place for documentation since it took me a second to think through these lines (and I wrote it 🙃).